README.md

```diff
@@ -530,7 +530,7 @@ Mailing lists servers and mass mailing software - one message to many recipients
 - [Friendica](https://friendi.ca/) - Social Communication Server. ([Source Code](https://github.com/friendica/friendica)) `AGPL-3.0` `PHP`
 - [Glosa](https://github.com/glosa/glosa-server) - Open source commentary system easy to integrate with static pages. You can import from Disqus. `GPL-3.0` `Java`
 - [GNU social](https://gnu.io/social/) - Social communication software for both public and private communications. ([Source Code](https://notabug.org/diogo/gnu-social)) `AGPL-3.0` `PHP`
-- [Gosora](https://gosora-project.com/) - Gosora is an ultra-fast and secure forum software written in Go that balances usability with functionality. ([Source Code](https://github.com/Azareal/Gosora)) `GPL-3.0` `Go`
+- [Gosora](https://github.com/Azareal/Gosora) - Gosora is an ultra-fast and secure forum software written in Go that balances usability with functionality. `GPL-3.0` `Go`
```

Comment:
I think it's safe to say Gosora is unmaintained, no activity since Jul 29, 2021, no answers on issue tracker, we should remove it. Will open another issue

Reply:
https://github.com/awesome-selfhosted/awesome-selfhosted/issues/3044